base_station/gui/src/components/PlaybackAutonData.vue

```diff
@@ -78,6 +80,9 @@ export default {
               case 'Odom bearing':
```

Comment:
nit pick, could these be enums instead of strings?

Reply:
No, they're strings from the csv file